test/puppetlabs/puppetdb/testutils/jetty.clj

```diff
@@ -44,22 +74,31 @@
 
 (defn puppetdb-instance
   "Stands up a puppetdb instance with `config`, tears down once `f` returns.
-   Adjusts *server* and *base-url* to refer to the instance."
+  If the port is assigned by Jetty, use *port* to get the currently running port."
```

Comment:
Suspect you might have accidentally clobbered the server/base-url docs with the older port docs here.